webserver/webserver/src/main/java/io/helidon/webserver/ServerThreadPoolSupplier.java

```diff
@@ -14,22 +14,37 @@
  * limitations under the License.
  */
 
-package io.helidon.common.configurable;
+package io.helidon.webserver;
```

Comment:
This is a backward incompatible change.
Also this may be used by any server, not just the webserver.

Reply:
The issue is the desire to ensure that when a task _is_ rejected by the pool, the result will be a `503` rather than a `500`. I wanted to make a generic change in `RequestRouting` that could handle any `HttpException`, wrapped or not, and still return the contained status. This approach requires that an `HttpException` should be thrown by the rejection handler. Given that this class was only recently introduced, the probability of breakage seems quite low, but is obviously non-zero.

I can think of two ways to put this class back in the original module:

1. Add a new exception (e.g. `ServerBusyException`) in the original module and make the corresponding (less generic) change in `RequestRouting`, or
2. Changing this class so that it does not default the rejection handler, and change the usage sites in `Server` and `JerseySupport` to pass one that throws the `HttpException`.

Reply:
Default rejection handler was in the original implementation as well - can we keep that the same? I would opt for option 2.

Reply:
I realized that there was no need to add a new exception class; I updated `RequestRouting` to handle `java.util.concurrent.RejectedExecutionException`. Moved the class back to the helidon-common-configurable module.